pkgs/by-name/vg/vgmtrans/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  autoPatchelfHook,
+  qt6,
+}:
+stdenv.mkDerivation {
+  pname = "vgmtrans";
+  version = "1.1-unstable-2024-07-31";
+
+  src = fetchFromGitHub {
+    owner = "vgmtrans";
+    repo = "vgmtrans";
+    rev = "9de4c06b03f8c9ec9ecb5c81dfcb53d1b8f75bbe";
+    hash = "sha256-689ceDbQrifH64eA79fCpa+eB70gJYKSyiZDlC1HZiA=";
+    fetchSubmodules = true;
```

Comment:
Please devendor the deps from https://github.com/vgmtrans/vgmtrans/blob/master/.gitmodules

Reply:
That would make the derivation a lot more complicated as it expects these libraries' source code to be installed at specific locations. Would try this later if I have time

Reply:
I mean we can symlink them in place but we certainly do not want to use vendored sources for zlib-ng.

Reply:
I mean we can symlink them in place but we certainly do not want to use vendored sources for zlib-ng.